Change PR template (#222)
vtex-apps/admin-pages: .github/PULL_REQUEST_TEMPLATE.md

#### What problem is this solving?

<!--- What is the motivation and context for this change? -->

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.

> - [ ] If applicable, link this PR to a Clubhouse story.

Reviewer: ```suggestion
- [ ] Link PR to a Clubhouse story (if applicable)
```
Começar com a ação para facilitar a checagem

> - [ ] Whenever possible, create tests for bug fixes.

Reviewer: This item doesn't imply a "yes/no" answer because it has both a conditional and optional characteristic in it ("whenever posssible" and "if bug fix")

> - [ ] If applicable, delete workspace after merge.

Reviewer: Tem que deletar workspace depois do merge?

Reviewer 2: I personally think it's a good practice to use disposable workspaces, in a one-per-PR basis. It allows you to work on multiple things at the same time without losing track of which is which.

#### What problem is this solving?

<!--- What is the motivation and context for this change? -->

#### How should this be manually tested?

[Workspace](url)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.

> - [ ] Link PR to a Clubhouse story (if applicable).

Reviewer 2: ```suggestion
- [ ] Linked this PR to a Clubhouse story (if applicable).
```

- [ ] Updated/created tests (important for bug fixes).

> - [ ] Delete workspace after merge (f applicable).

Reviewer 2: ```suggestion
- [ ] Deleted the workspace after merging this PR (if applicable).
```
